CONTRIBUTING.md

```diff
@@ -61,7 +61,7 @@ conda activate srf
 ```
 #### Build SRF
 ```bash
-mkdir $SRF_HOME/build
+mkdir SRF_HOME/build
```

Comment:
Why did you remove the `$`? And leave it in the next line?

Reply:
I was moving to fast and changed the wrong line. Fixed.